lib/ash/actions/read/relationships.ex

```diff
@@ -176,6 +176,7 @@ defmodule Ash.Actions.Read.Relationships do
               actor: source_query.context[:private][:actor],
               tracer: source_query.context[:private][:tracer]
```

Comment:
We should set it here in the options list actually.

Reply:
source_query.context[:private] doesn't have tenant, so I had to set it with source_query.tenant, which looks a bit awkward.

Reply:
Thats fine :)